src/lib/stack/_mixins.scss

```diff
@@ -0,0 +1,44 @@
+@use '../theme';
+@use './variables';
+
+@mixin core-styles() {
+  .forge-stack {
+    @include base();
+    @include theme.css-custom-property(gap, --forge-stack-gap, variables.$gap);
+    @include theme.css-custom-property(height, --forge-stack-height, 100%);
+  }
+}
+
+@mixin base() {
```

Comment:
I think since the goal of this component to is to only lay out items with a gap between them in specified orientation that it will need either `flex` or `grid`?

Reply:
I think since the goal of this component to is to only lay out items with a gap between them in specified orientation that it will need either `flex` or `grid`?

Reply:
Ah, yep, definitely agree!